### PR TITLE
Completed the CRUD for Notices

### LINKED ISSUE #37: complete CRUD for Notice route
### Description
<write your description> Update and Delete operations need to be implemented

### PR DESCRIPTION
**Pull request by ankiiitraj**
fixes #37 
<Do not delete this template>
Description: <provide a description> Created and modified Crud operation for notices.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes that were demanded in the linked issue.
- [ ] I've tested my code on a Chromium-based browser.
- [ ] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
N/A

### CODE REVIEW

File: backend/routes/notice.js [hunk 1]
```diff
@@ -4,6 +4,10 @@ var router = express.Router();
 const auth = require("../util/auth");
 const Notice = require("../models/notice.model");
 
+<<<<<<< HEAD
```
Comment (EmperorYP7) [inline backend/routes/notice.js]: oops. Forgot to resolve this.

File: backend/routes/notice.js [hunk 2]
```diff
@@ -23,19 +27,62 @@ router.post("/", auth, function (req, res, next) {
     })
     .catch((err) => {
       res.status(500).json({
+<<<<<<< HEAD
```
Comment (EmperorYP7) [inline backend/routes/notice.js]: Here as well.

File: backend/routes/notice.js [hunk 3]
```diff
@@ -23,19 +24,55 @@ router.post("/", auth, function (req, res, next) {
...
+    .catch((err) => {
+      console.log(err);
+      res.status(500).json({
+        message: "Something went wrong",
+      });
+    });
+});
+
+// Returns all the notices
+router.get("/all", (req, res, next) => {
```
Comment (EmperorYP7) [inline backend/routes/notice.js:53]: I wonder where we'll be using this.. :thinking:

**Review by EmperorYP7 — APPROVED** [review]
LGTM!

File: backend/routes/notice.js (hunk 3 above)
Comment (anktrj) [reply backend/routes/notice.js:53]: I'm also not sure of this, we'll be allowing all previous notices or not!

Comment (EmperorYP7) [reply backend/routes/notice.js:53]: No harm in having an option to retrieve all of em..

**Pull request merged**